aireview/src/aireview/domain/entities/review_comment.py

```diff
@@ -4,6 +4,7 @@
 
 @dataclass
```

Comment:
**documentation**: Missing class-level docstring explaining the purpose and usage of ReviewComment class.